src/specBuilder/combo/comboSpecBuilder.ts

```diff
@@ -99,7 +99,8 @@ export const getComboChildName = (cur: ComboChildElement, comboName: string, ind
 		return cur.props.name;
 	}
 	const displayName = getDisplayName(cur);
-	return combineElementNames(comboName, `${displayName}${index}`);
+	const combinedElementName = combineElementNames(comboName, `${displayName}${index}`);
+	return combinedElementName
```

Comment:
```suggestion
	return combineElementNames(comboName, `${displayName}${index}`);
```

This was probably added for debugging.